packages/create-llama/questions.ts

```diff
@@ -417,6 +418,22 @@ export const askQuestions = async (
     }
   }
 
+  if (program.framework === "nextjs") {
```

Comment:
No support for express?

Reply:
You're right, added as well. We also have a Python SDK, should I also add fastapi? (basically, removing the `if` altogether)

Reply:
Sure, more than welcome to add support for Python too!, But better make it in a second PR as we're changing the Python code right now